Algorithmia/datafile.py

```diff
@@ -52,6 +52,11 @@ def getFile(self, as_path=False):
             return open(f.name)
 
     def getAsZip(self):
+        """Download/decompress file/directory and return path to file/directory.
+        
+        Expects the `DataFile` object to point to a zip-compatible data API location.
```

Comment:
as discussed in Slack let's clarify this wording:
`Expects the `DataFile` object to contain a data API path pointing to a file/directory compressed with a zip-based compression algorithm.`